contents/stats_distributions.md

```diff
@@ -2,16 +2,12 @@
 
 Before defining distributions, we must talk about **probability**.
 
-**The probability of an event is a real number between 0 and 1, where 0 indicates the impossibility of the event and 1 indicates the certainty of the event**.
+In the classical definition of probability, **the probability of an event is a real number between 0 and 1, where 0 indicates the impossibility of the event and 1 indicates the certainty of the event**.
```

Comment:
```suggestion
**The probability of an event is a real number $p$ where $0 \leq p \leq 1$.
Here, 0 indicates the that an event is impossible and 1 indicates that an event will certainly happen.**
```
One of the critiques was that "between 0 and 1" is not the same as "the closed interval between 0 and 1", that is, $[a, b] = \{ x \in \mathbb{R} | a \leq x \leq b \}$.

Open and closed intervals cannot be expressed clearly in the English language, so that's why I suggest using $x$ where $a \leq x \leq b$ because that notation requires less knowledge from the reader than $[a, b]$.